scisample/uqpipeline_sampler.py

```diff
@@ -0,0 +1,172 @@
+"""
+Module defining the UQ Pipeline sampler object.
+"""
+
+import logging
+import random
+import os
+import sys
+from contextlib import suppress
+
+UQPIPELINE_SAMPLE = False
+UQPIPELINE_SAMPLE_PATH = '/collab/usr/gapps/uq/UQPipeline/smplg_cmpnt'

```

Comment:
Consider adding a way to configure this, maybe as part of the sampler  data, in case someone is using development tools.

Reply:
I can add a configuration variable. I plan to have default paths as determined by /etc/home.config. I will also test whether the uqpipeline sampling tools work for gpu machines. 

In the long term, I was told that the uq_pipeline team is planning to open source some of their code, so this configuration won't be needed.